src/media/js/apps_buttons.js

```diff
@@ -47,29 +48,23 @@ define('apps_buttons',
 
         // If it's a paid app, ask the user to sign in first.
         if (product.receipt_required && !user.logged_in()) {
-            var payWindow;
+            console.log('Install suspended; user needs to log in');
 
-            if (!capabilities.navPay) {
-                console.log('Creating payment window ahead of time via user click');
-                payWindow = payments.utils.openWindow({url: ''});
-            }
+            // Create a blank window here so we can pass it to the login func;
+            loginPopup = (!capabilities.navPay) ? utils.openWindow() : undefined;
```

Comment:
nit: I think ``payWindow`` is a better name for this variable since the window will be used for payment; the pay flow just happens to require a login first.